index.html

```diff
@@ -71,14 +70,18 @@
     <br/>
     <div class="ui container">
         <div class="ui stackable grid">
-            <div class="twelve wide column">
-                <p> Lorem ipsum dolor sit amet, pri ex animal labores, vero lobortis ex pri, dicit ridens has ne. Unum verear
-                    suscipit id duo. Habemus quaestio quaerendum has at. Eu nam eirmod assueverit sadipscing.</p>
-                <p> Dolor fastidii perfecto id quo, quidam utamur sit ea. Eum etiam postea no. Quo no partem indoctum deseruisse,
-                    pri harum ceteros at, detraxit sensibus no sea. Cu duo stet erant, ius adhuc utamur eleifend ne, ea mei
-                    soleat iriure. Mel enim soluta interpretaris ex, admodum persecuti nam ea.</p>
+            <div class="sixteen wide column">
                     <h1> Slideshow goes here </h1>
             </div>
+            <div class="four wide column"> <image src="assets/images/principal-welcome.jpg"></image> </div>
```

Comment:
add a class `class="ui fluid image"` to the `<image>` tag in order to keep it from overlapping with the text.